apps/daimo-mobile/src/view/screen/AccountScreen.tsx

```diff
@@ -165,6 +165,13 @@ function AccountScreenBody({
     });
   }, [nav, eAcc, account]);
 
+  const goToRequest = () => {
+    nav.navigate("HomeTab", {
+      screen: "Receive",
+      params: { autoFocus: true, recipient: { type: "eAcc", ...eAcc } },
+    });
+  };
```

Comment:
just for i recommend matching `send` above here. can change it to


```ts
// Generalize addLastSendRecvTime to include both send & recv time
const recipient = addLastTransferTimes(account, eAcc);

const canSend = canSendTo(eAcc);
const goToSend = () => ...

const canRequest = canRequestFrom(eAcc);
const goToRequest = () => ...
```

... canRequestFrom can check whether eAcc is a Daimo account.